<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsWindow</class>
 <widget class="QWidget" name="StatsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1323</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_execute">
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="JsScriptMultiSelector" name="js_script_multi_selector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="AudioFileMultiSelector" name="audio_file_multi_selector" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>155</number>
       </attribute>
       <column>
        <property name="text">
         <string>Script name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Mean error</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Median error</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Max error</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Max error file</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Invalid</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JsScriptMultiSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/js_script_multi_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AudioFileMultiSelector</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/audio_file_multi_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
